cwe_api/inferencer.py

```
@@ -0,0 +1,50 @@
+
+ID2LABEL_PATH = './id2label.json'
+LABEL2ID_PATH = './label2id.json'
+MODEL_PATH = "./modelo_cwe/checkpoint-141693"
+NUMBER_OF_PREDICTIONS = 3
+
+class BestCweClassifications(TextClassificationPipeline):
+    def postprocess(self, model_outputs):
+        best_class = model_outputs["logits"]
+        return best_class
+
+def inferencer(vuln):
```

Ruff

Missing return type annotation for public function `inferencer`

(ANN201)